students/Chao/session02/series.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+"""
+IntroPython-2017
+Session02
+Fibonacci and Lucas Series Exercises
+"""
+
+def fib(n):
+    """
+    nth Fibonacci number implementation with recursion
+    """
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    return fib(n-1) + fib(n-2)
+
+def new_fib(n):
+    """
+    A new Fibonacci number implementation
+    Calling sum series function with start numbers 0 and 1
+    No optional parameter input here since default sum series function works as fib
+    """
+    return(sum(n))
+
+
+def luc(n):
+    """
+    nth Lucas number implementation with recursion
+    """
+    if n == 0:
+        return 2
+    elif n == 1:
+        return 1
+    return luc(n-1) + luc(n-2)
+
+def new_luc(n):
+    """
+    A new Lucas number implementation
+    Calling sum series function with start numbers 2 and 1
+    """
+    return(sum(n, 2, 1))
+
+def sum(n, a=0, b=1):
```

Comment:
sum is a built in name -- you really don't want to use it!

but the code is great :-)